src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -3502,6 +3686,176 @@ BC30469: Reference to a non-shared member requires an object reference.
             VerifyOperationTreeAndDiagnosticsForTest(Of UnaryExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DelegateCreation_ParenthesizedAddressOf_CType()
+            Dim source = <![CDATA[
+Imports System
+
+Public Class C
+    Public Sub M1()
+        CType(((AddressOf M2)), Action)'BIND:"CType(((AddressOf M2)), Action)"
+    End Sub
+
+    Public Sub M2()
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IDelegateCreationOperation (OperationKind.DelegateCreation, Type: System.Action) (Syntax: 'CType(((Add ... )), Action)')
+  Target: 
+    IParenthesizedOperation (OperationKind.Parenthesized, Type: System.Action) (Syntax: '((AddressOf M2))')
```

Comment:
>IParenthesizedOperation (OperationKind.Parenthesized, Type: System.Action) (Syntax: '((AddressOf M2))') [](start = 4, length = 103)

Why there is no type for IParenthesizedOperation?

Reply:
I don't understand: are you saying that the IParenthesizedOperation should not have a type?

---
In reply to: [152434373](https://github.com/dotnet/roslyn/pull/23315#discussion_r152434373) [](ancestors = 152434373)

Reply:
> I don't understand: are you saying that the IParenthesizedOperation should not have a type? 

Sorry, that is probably the issue I wanted to bring up, but probably got distracted before typing it in.